### DISCUSSION TITLE
Parallel pack simulations

### DISCUSSION THREAD

**Discussion by jonasbrondum** [pybamm-team/liionpack#306] [Q&A]
Hi,

I am new to the package and have been looking for documentation for how to simulate a parallel pack i.e., a simulation of multiple modules in parallel.
So creating two separate modules of let's say 20 cells in parallel with 5 in series. We would in total have 200 cells distributed between to separate packs. Is it possible with the current package to simulate such a situation?

Best, Jonas

**Comment by TomTranter** [comment]
If you can build the netlist in LTspice or otherwise it should be possible. There is an example circuit here liionpack/circuits/4p1s.cir
